app/vmselect/main.go

```diff
@@ -119,6 +120,14 @@ func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 		}
 	}
 
+	ctx := tracer.NewContext(r)
+	defer func() {
+		ctx.Done(func() string {
+			return fmt.Sprintf("query trace %q", ctx.ID())
+		})
+		fmt.Println(ctx.Print())
```

Comment:
I'm not sure, that Println must be used, it breaks logs parsing. Is it possible to use standard logger?